web-server/src/components/Settings/SyncDaysSetting.tsx

```diff
@@ -0,0 +1,146 @@
+import { FlexBox } from '../FlexBox';
+import { Box, Button, Divider, TextField } from '@mui/material';
+import { LoadingButton } from '@mui/lab';
+import { Line } from '../Text';
+import { FC, useEffect, useState } from 'react';
+import { useAuth } from '@/hooks/useAuth';
+import { useSnackbar } from 'notistack';
+import { useDispatch, useSelector } from '@/store';
+import {
+  getDefaultSyncDaysSettings,
+  updateDefaultSyncDaysSettings
+} from '@/slices/org';
+import { FetchState } from '@/constants/ui-states';
+
+export const SyncDaysSetting: FC = () => {
+  const dispatch = useDispatch();
+  const { orgId } = useAuth();
+  const { enqueueSnackbar } = useSnackbar();
+  const initialSyncDays = useSelector((state) => state.org.defaultSyncDays);
+  const [syncDays, setSyncDays] = useState<number>(initialSyncDays);
+  const isSaving = useSelector(
+    (s) => s.org.requests?.defaultSyncDays === FetchState.REQUEST
+  );
+  const [error, setError] = useState('');
+
+  const getSyncDays = async (): Promise<number> => {
+    const res = await dispatch(getDefaultSyncDaysSettings({ orgId }));
+    return res.payload as number;
+  };
+
+  const updateSyncDays = async (days: number): Promise<number> => {
+    const res = await dispatch(
+      updateDefaultSyncDaysSettings({ orgId, defaultSyncDays: days })
+    );
+    return res.payload as number;
+  };
+
+  const handleDiscard = () => {
+    setSyncDays(initialSyncDays);
+    setError('');
+  };
+
+  const handleSave = async () => {
+    try {
+      await updateSyncDays(syncDays);
+      enqueueSnackbar('Default Sync Days Updated Successfully', {
+        variant: 'success',
+        autoHideDuration: 3000
+      });
+    } catch (e) {
+      handleDiscard();
+      return enqueueSnackbar('Default Sync Days Update Failed', {
+        variant: 'error',
+        autoHideDuration: 2000
+      });
+    }
+  };
+
+  useEffect(() => {
+    if (!orgId) return;
+    getSyncDays().then((d) => setSyncDays(d));
+  }, []);
+
+  return (
+    <FlexBox col gap2>
+      <FlexBox big bold gap1 component={Line} white sx={{ fontSize: '20px' }}>
+        Organization Settings
+      </FlexBox>
```

Comment:
fontSize='20px' no effect, but 'huge' instead of 'big' seems ok.

Reply:
I just wanted to say that instead of using Flexbox, you can easily use <Line>. The props are similar, just use big, small, huge or whichever font size fits the best!